### PR TITLE
ci: performance tests

### PR DESCRIPTION
**Pull request by akhdrv**
:baby_chick: Download canary assets:

[colors--canary.274.3806650475.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.274.3806650475.xml)  
[color_metro_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.274.3806650475.swift)  
[color_metro_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.274.3806650475.kt)  
[color_metro_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.274.3806650475.ts)  
[color_sbermarket_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.274.3806650475.swift)  
[color_sbermarket_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.274.3806650475.kt)  
[color_sbermarket_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.274.3806650475.ts)  
[color_sberprime_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.274.3806650475.swift)  
[color_sberprime_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.274.3806650475.kt)  
[color_sberprime_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.274.3806650475.ts)  
[color_selgros_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.274.3806650475.swift)  
[color_selgros_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.274.3806650475.kt)  
[color_selgros_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.274.3806650475.ts)  
[color_smbusiness_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.274.3806650475.swift)  
[color_smbusiness_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.274.3806650475.kt)  
[color_smbusiness_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.274.3806650475.ts)  
[PlasmaTokensColor--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.274.3806650475.swift)  
[typo_mage_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.274.3806650475.swift)  
[typo_mage_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.274.3806650475.kt)  
[typo_mage_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.274.3806650475.ts)  
[typo_plasma_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.274.3806650475.swift)  
[typo_plasma_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.274.3806650475.kt)  
[typo_plasma_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.274.3806650475.ts)  
[typo_ruler_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.274.3806650475.swift)  
[typo_ruler_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.274.3806650475.kt)  
[typo_ruler_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.274.3806650475.ts)  
[typo_sage_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.274.3806650475.swift)  
[typo_sage_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.274.3806650475.kt)  
[typo_sage_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.274.3806650475.ts)  
[typo_sbermarket_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.274.3806650475.swift)  
[typo_sbermarket_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.274.3806650475.kt)  
[typo_sbermarket_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.274.3806650475.ts)  
[typo_soulmate_ios-swift--canary.274.3806650475.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.274.3806650475.swift)  
[typo_soulmate_kotlin--canary.274.3806650475.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.274.3806650475.kt)  
[typo_soulmate_react-native--canary.274.3806650475.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.274.3806650475.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.121.1-canary.274.3806650475.0
  npm install @salutejs/plasma-ui@1.154.1-canary.274.3806650475.0
  npm install @salutejs/perftool@0.2.1-canary.274.3806650475.0
  # or 
  yarn add @salutejs/plasma-temple@1.121.1-canary.274.3806650475.0
  yarn add @salutejs/plasma-ui@1.154.1-canary.274.3806650475.0
  yarn add @salutejs/perftool@0.2.1-canary.274.3806650475.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🌚 Missing base report**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

File: .github/workflows/performance-test-main.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        with:
+          path: node_modules
+          key: npm-deps-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            npm-deps-${{ hashFiles('package-lock.json') }}
+
+      - name: Setup packages
+        run: |
+          npm ci
+          npx lerna bootstrap --scope "@salutejs/plasma-ui"
```
Comment (KateKate) [inline .github/workflows/performance-test-main.yml:37]: а ты не хочешь тут и perftool из репы брать? npx lerna bootstrap --scope "@salutejs/plasma-ui" --scope "@salutejs/perftool"
нам тогда perftool вообще можно не паблишить на npm

File: .github/workflows/performance-test-main.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    runs-on: ubuntu-latest
+    env:
+      NPM_REGISTRY_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare repository
+        run: git fetch --unshallow --tags
+
+      - name: Use Node.js 12.x
```
Comment (KateKate) [inline .github/workflows/performance-test-main.yml:21]: почему тут тоже не может Node 16?

File: utils/perftool/src/config/common.ts [hunk 3]
```diff
@@ -58,31 +58,39 @@ export type CliConfig = Partial<
...
+    }
+
+    return value;
+}
+
 export function getConfig(cliConfig: CliConfig = {}, projectConfig: ProjectConfig = {}): Config {
     debug('getting final config');
     const mixedInputConfig = { ...cliConfig, ...projectConfig };
 
     const result = {
```
Comment (KateKate) [inline utils/perftool/src/config/common.ts:73]:
```js
const DEFAULT_CONFIG = {
 taskConfiguration: {},
 tasks: [],
 ...
 };
 
 const result = {
   ...DEFAULT_CONFIG,
   ...сliConfig,
   ...projectConfig,
 };
 
1) будет понятнее, что за дефолтный конфиг
2) проверку withDefault не надо будет для каждого ключа вызывать

File: .github/workflows/performance-test-main.yml (hunk 1 above)
Comment (akhdrv) [reply .github/workflows/performance-test-main.yml:37]: обсудили, мешает sh скрипт, который генерирует лерна при линкинге

File: .github/workflows/performance-test-main.yml (hunk 2 above)
Comment (akhdrv) [reply .github/workflows/performance-test-main.yml:21]: npm начнет перегенеривать локфайл, что долго, npm ci не сработает

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**
boolean
Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🔴 Fail**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-274/

File: utils/perftool/src/config/common.ts (hunk 3 above)
Comment (akhdrv) [reply utils/perftool/src/config/common.ts:73]: в целевом решении будет deep merge, так что это все равно поменяется. тут только пофиксил баг с логикой проставления дефолтных значений

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-274/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-274/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-274/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-274/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-274/temple-storybook/

**Review by KateKate — APPROVED** [review]

File: .github/workflows/performance-test-main.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      - name: Run performance test
+        run: |
+          npm run perftool:test --prefix="./packages/plasma-ui"
+
+      - name: Upload performance test results to S3
+        uses: jakejarvis/s3-sync-action@master
+        with:
+          args: --acl public-read --follow-symlinks --delete
+        env:
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
```
Comment (Yeti-or) [inline .github/workflows/performance-test-main.yml:53]: @kayman233 а там с местом всё ок?
@akhdrv сколько у нас будут отчеты отжирать?

Comment (akhdrv) [reply .github/workflows/performance-test-main.yml:53]: сейчас килобайты, в перспективе единицы мегабайт. в общем, скромно

Comment (akhdrv) [reply .github/workflows/performance-test-main.yml:53]: скорее даже не единицы мегабайт, а сотни килобайт, если говорить чисто про json отчеты

Comment (kayman233) [reply .github/workflows/performance-test-main.yml:53]: инфра говорит, что норм, если просто будем складывать отчеты, но при этом "главное чтобы как-то потом на них через сайт плазмы не попасть"

Comment (akhdrv) [reply .github/workflows/performance-test-main.yml:53]: А в чем проблема доступа с сайта плазмы? Отчеты такие же публичные как и сама плазма

**Review by Yeti-or — APPROVED** [review]

**Review by Yakutoc — APPROVED** [review]

**Pull request merged**